metaserver/mongoDB.go

```diff
@@ -0,0 +1,471 @@
+package metaserver
+
+import (
+	"skybin/core"
+
+	"github.com/globalsign/mgo"
+)
+
+const dbName = "skybin"
+const dbAddress = "127.0.0.1"
+
+type mongoDB struct{}
```

Comment:
Recommendation: Store connected DB within this struct. Do not reconnect to the database on each call to getMongoCollection().

Reply:
That's a good idea. For some reason I was thinking Mongo's consistency model relies on sessions, but, double checking the docs, that only happens if you sent MongoDB to causal consistency mode. Since we're using strong consistency, we won't have a problem with that.

Reply:
I moved the session information to the DB struct. However, we still want to create a new session for every call, because it allows the driver to take advantage of concurrency.

See [this StackOverflow question](https://stackoverflow.com/questions/26574594/best-practice-to-maintain-a-mgo-session) for details